### PR TITLE
[FEATURE] #80: Added configuration for a user to setup desired columns

### PR DESCRIPTION
**Pull request by plessmann**
Added the possibility for a user to hide columns of the timesheet like the wage, customer etc. 
Its not always desired to see all the columns when you sit in an office next to other employees. Sometime you do not want to show for which other clients you have worked in advance. With this feature you can decide yourself which columns you really need and want to display. It configurable per user. Language strings for EN, DE, FR, ES, IT, PL have been added.

2015-09-12 16_30_09-lars - kimai

2015-09-12 16_41_25-lars - kimai

### CODE REVIEW

File: core/extensions/ki_timesheets/templates/scripts/main.php [hunk 1]
```diff
@@ -12,39 +12,35 @@
...
+if($this->visibleColumn['from']) echo "                <td class=\"from\">" . $this->kga['lang']['in'] . "</td>\n";
+if($this->visibleColumn['to']) echo "                <td class=\"to\">" . $this->kga['lang']['out'] . "</td>\n";
+if($this->visibleColumn['time']) echo "                <td class=\"time\">" . $this->kga['lang']['time'] . "</td>\n";
+if($this->visibleColumn['wage']) echo "                <td class=\"wage\">" . $this->kga['lang']['wage'] . "</td>\n";
+if($this->visibleColumn['customer']) echo "                <td class=\"customer\">" . $this->kga['lang']['customer'] . "</td>\n";
+if($this->visibleColumn['project']) echo "                <td class=\"project\">" . $this->kga['lang']['project'] . "</td>\n";
+if($this->visibleColumn['activity']) echo "                <td class=\"activity\">" . $this->kga['lang']['activity'] . "</td>\n";
+if($this->visibleColumn['trackingnumber']) echo "                <td class=\"trackingnumber\">" . $this->kga['lang']['trackingNumber'] . "</td>\n";
+if($this->visibleColumn['username']) echo "                <td class=\"username\">" . $this->kga['lang']['username'] . "</td>\n";
+?>
```
Comment (simonschaufi) [inline core/extensions/ki_timesheets/templates/scripts/main.php:43]: can you please write it like this: `echo '<td class="..."'` and remove the `\n` at the end.

File: core/extensions/ki_timesheets/templates/scripts/timeSheet.php [hunk 2]
```diff
@@ -8,21 +8,19 @@
...
+if($this->visibleColumn['from']) echo "                <col class=\"from\" />\n";
+if($this->visibleColumn['to']) echo "                <col class=\"to\" />\n";
+if($this->visibleColumn['time']) echo "                <col class=\"time\" />\n";
+if($this->visibleColumn['wage']) echo "                <col class=\"wage\" />\n";
+if($this->visibleColumn['customer']) echo "                <col class=\"customer\" />\n";
+if($this->visibleColumn['project']) echo "                <col class=\"project\" />\n";
+if($this->visibleColumn['activity']) echo "                <col class=\"activity\" />\n";
+if($this->visibleColumn['trackingnumber']) echo "                <col class=\"trackingnumber\" />\n";
+if($this->visibleColumn['username']) echo "                <col class=\"username\" />\n";
+?>
```
Comment (simonschaufi) [inline core/extensions/ki_timesheets/templates/scripts/timeSheet.php:23]: can you please write it like this: `echo '<col class="..."'` and remove the `\n` at the end.

**Comment by simonschaufi** [comment]
@plessmann update?

**Comment by plessmann** [comment]
Still needed? @simonschaufi

**Comment by simonschaufi** [comment]
well, if you change what I requested to change then I will be happy to merge it.

**Pull request closed without merging**